buildscripts/docker/docker_release/Dockerfile

```diff
@@ -0,0 +1,37 @@
+# To test: docker build -t test_dockerhub .
+FROM mambaorg/micromamba:latest AS builder
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+ENV JUPYTER_ENABLE_LAB=yes
+
+COPY --chown=$MAMBA_USER:$MAMBA_USER ./buildscripts/docker/docker_release/env.yaml /tmp/env.yaml
+
+RUN micromamba install  -y -n base -f /tmp/env.yaml && \
+    micromamba clean --all --yes
+
+FROM ubuntu:latest
+
+USER root
+# Copy the conda environment from the builder stage
+COPY --chown=root --from=builder /opt/conda /opt/conda
+
+# Install necessary packages
+RUN apt-get update && \
+    apt-get install -y bzip2 git make python3 python3-pip vim wget && \
```

Comment:
Why do we need these packages? Particularly python, shoudn't we be using conda's python?

Reply:
Not sure, This was in the PR I took over. 
I only added `vim` as I needed that when I was testing.

Reply:
Ok I think we need to at least try removing them, we want our images to be as minimal as possible